be/src/common/config.h

```diff
@@ -660,7 +660,7 @@ CONF_Int64(pipeline_prepare_thread_pool_queue_size, "102400");
 // The buffer size of SinkBuffer.
 CONF_Int64(pipeline_sink_buffer_size, "64");
 // The degree of parallelism of brpc.
-CONF_Int64(pipeline_sink_brpc_dop, "8");
+CONF_Int64(pipeline_sink_brpc_dop, "64");
```

Comment:
is this change related to doc? for example the default value in a config file.

Reply:
> is this change related to doc? for example the default value in a config file.

I did't find it in our user docs and default config file, this seems to be an advaned configuration